src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/IncentiveMappingResource.kt

```diff
@@ -53,6 +53,16 @@ class IncentiveMappingResource(private val mappingService: IncentiveMappingServi
         description = "Unauthorized to access this endpoint",
         content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
       ),
+      ApiResponse(
+        responseCode = "409",
+        description = "Indicates a duplicate incentive has been rejected. If Error code = 1409 the body will return a DuplicateErrorResponse",
```

Comment:
error code `1409`?

Reply:
yes - its just a made up code to identify this type of 409